matching/pom.xml

```diff
@@ -59,6 +67,16 @@
       <version>1.4</version>
       <classifier>linux64</classifier>
     </dependency>
+    <dependency>
+      <groupId>com.diffplug.spotless</groupId>
+      <artifactId>spotless-maven-plugin</artifactId>
+      <version>${spotless.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.scalameta</groupId>
+      <artifactId>scalafmt-core_${scala.majorVersion}</artifactId>
+      <version>${scalafmt.version}</version>
+    </dependency>
```

Comment:
Same thing here - dependencies will automatically be pulled in from the parent POM and don't need to be redeclared